Keas.Mvc/Controllers/FeedController.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Keas.Core.Data;
+using Keas.Mvc.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Keas.Core.Domain;
+
+namespace Keas.Mvc.Controllers
+{
+    public class FeedController : SuperController
+    {
+         private readonly ApplicationDbContext _context;
+
+        public FeedController(ApplicationDbContext context)
+        {
+            this._context = context;
+        }
+        public async Task<IActionResult> TeamFeed(Guid id, string space)
+        {            
+            if(space == "yes")
```

Comment:
@jcubbage break these two paths into different methods so everything isn't in the one method

Reply:
I origanally had that, but the issue wanted it as one with a parameter :)

Reply:
@jcubbage The one parameter is fine and can stay the same, I just think you should make a private method or two to handle the actual logic of it.